config/routes.rb

```diff
@@ -304,10 +297,91 @@
   get "signup" => "static#show", id: "signup", constraints: { format: /(json|html)/ }
   get "login-preferences" => "static#show", id: "login", constraints: { format: /(json|html)/ }
 
+  get "my/*path", to: 'users#my_redirect'
+  get "user_preferences" => "users#user_preferences_redirect"
+
+  # New /u/ routes
```

Comment:
The problem is many routes are `put` or `post`, or have aliases. I'd basically have to build a mini hash structure to account for all of that, which I'm not sure is much better than just copying and pasting with a comment.